man/loader.conf.xml

```diff
@@ -399,6 +399,14 @@ sbvarsign --attr "${attr}" --key KEK.key --cert KEK.pem --output db.auth db db.e
 
         <xi:include href="version-info.xml" xpointer="v251"/></listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term>reboot-on-error</term>
+
+        <listitem><para>Takes a boolean argument. Enable or disable (the default) auto reboot in case the selected entry fails to start.</para>
```

Comment:
this is a bit confusing. i.e. i guess `(the default)` is supposed to apply to the `disable` left of it. But it could be misunderstood as if it applies to `auto reboot` to the right of it. Which would have opposing meaning.

Maybe just add another sentence: Defaults to off.